.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,16 @@
+# normalize
+- id: schemalint-normalize
+  name: normalize JSON Schema
+  description: Normalize JSON schema files
+  entry: schemalint normalize
+  language: golang
+  files: \.json$
```

Comment:
AFAIK this will run for any JSON file in a repo and that's not what we want. We have to limit it to just 1 file named `values.schema.json`